scripts/backend_test_shards.json

```diff
@@ -303,7 +303,10 @@
         "core.platform.storage.cloud_storage_emulator_test",
         "core.platform.storage.dev_mode_storage_services_test",
         "core.platform.storage.cloud_storage_services_test",
-        "core.domain.contribution_stats_services_test"
+        "core.domain.contribution_stats_services_test",
+        "core.domain.voiceover_regeneration_services_test",
+        "core.platform.azure_speech_synthesis.azure_speech_synthesis_services_test",
+        "core.platform.azure_speech_synthesis.dev_mode_azure_speech_synthesis_services_test"
```

Comment:
This shard is [failing](https://github.com/oppia/oppia/actions/runs/12960333954/job/36154031659?pr=21599), so I can't check its runtime. Please fix the backend test failures.

Reply:
Hi @U8NWXD now the tests are passing PTAL. Thank you!

Reply:
Backend shard 4 is now taking 37 mins to complete, which exceeds our guideline of 30 mins (https://github.com/oppia/oppia/wiki/Backend-tests#adding-new-tests-to-shards). Can you move these tests to shard 5, which is running in only 14 mins?

Reply:
Done

Reply:
backend tests failed

Reply:
The tests are now passing. PTAL